clang/test/SemaSYCL/global_variable_allowed.cpp

```diff
@@ -0,0 +1,62 @@
+// RUN: %clang_cc1 -fsycl-is-device -std=c++17 -sycl-std=2020 -verify -fsyntax-only %s
+
+#include "Inputs/sycl.hpp"
```

Comment:
Please add a comment about what test is doing. Thank you.

Reply:
Done.